WalletWasabi.Fluent/Behaviors/BindableFlyoutOpenBehavior.cs

```diff
@@ -33,11 +33,6 @@ protected override void OnAttached(CompositeDisposable disposable)
 		this.WhenAnyValue(x => x.IsOpen)
 			.Subscribe(isOpen =>
 			{
-				if (AssociatedObject is null)
```

Comment:
Can AssociatedObject be null in any case when this behavior is created?

Reply:
Because if that is surely not null, then there is another null check above that can be removed. Another question is if AssociatedObject can be changed meanwhile to null?

Reply:
This null check was written before the one above so I assume only the one above is necessary.
@danwalmsley ?